pkg/common/config/aws_config.go

```diff
@@ -6,8 +6,8 @@ import (
 )
 
 var (
-	// AWSAccount is the AWS account
-	AWSAccount = "config.aws.account"
+	// AWSAccountId is the AWS account (Env var: AWS_ACCOUNT_ID)
+	AWSAccountId = "config.aws.account.id"
```

Comment:
```suggestion
	AWSAccountId = "config.aws.accountID"
```

If we keep it as `config.aws.account.id`, a custom config would be:

```
config:
  aws:
    account:
      id: my-account-id
```

Where if we switch it to be `config.aws.accountID` a custom config would be:

```
config:
  aws:
    accountID: my-account-id
```

Let me know what you think!